app/domains/fluggastrechte/components/ActualArrivalCards.tsx

```diff
@@ -26,14 +26,17 @@ function ActualArrivalCards({
   readonly translations: Translations;
 }) {
   if (userData.bereich === "annullierung") {
-    return (
-      <SummaryDataOverviewCard
-        title="Angebotene Ersatzverbindung"
-        buttonUrl={`${FLOW_ID}/flugdaten/ersatzverbindung-daten`}
-        data={getAnnullierungData(userData)}
-        translations={translations}
-      />
-    );
+    if (userData.ersatzflug === "yes") {
+      return (
+        <SummaryDataOverviewCard
+          title="Angebotene Ersatzverbindung"
+          buttonUrl={`${FLOW_ID}/flugdaten/ersatzverbindung-daten`}
+          data={getAnnullierungData(userData)}
+          translations={translations}
+        />
+      );
+    }
+    return <></>;
```

Comment:
```suggestion
    return null;
```
https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null

I assume if we return Fragment, it will render an empty div